site/docs/components/country-symbol/examples.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Country Symbol
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="country-symbol" exampleName="BasicUsage" title="Basic Usage">
```

Comment:
i think `title` is no longer a valid prop with the updates merged early this week. It's now renamed to `displayName`